applications/surgical_scene_recon/run_complete_workflow.py

```
@@ -0,0 +1,382 @@
+            else:
+                print(f"\n❌ No checkpoint found")
+                return None
+                
+    except Exception as e:
+        print(f"\n❌ Training failed: {e}")
+        import traceback
+        traceback.print_exc()
+        return None
+
+
+def run_inference(data_dir, checkpoint_path, num_frames=-1, visualize=True):
```

Ruff (0.14.4)

Unused function argument: `num_frames`

(ARG001)